docs/development.md

```diff
@@ -25,3 +25,8 @@ $ docker stop wikijump
 ```
 
 Alternatively, you can install to your local system using `install.sh`. This may require tinkering depending on your exact platform and environment.
+
+
+You will likely want to set `allow_http` in `/var/www/conf/wikijump.ini` to `false`, since HTTPS locally is troublesome.
```

Comment:
Is this accurate? I feel like this should be either allow_http=true or allow_https=false

Reply:
`allow_https` doesn't exist, I guess it was a typo from wikijump-infra

Should be
```suggestion
You will likely want to set `allow_http` in `/var/www/conf/wikijump.ini` to `true`, since HTTPS locally is troublesome.
```